eslint.config.mjs

```diff
@@ -0,0 +1,9 @@
+import globals from "globals";
```

Comment:
Can you make sure the code style is the same as everywhere else?

* Single quotes vs double quotes
* With or without `;`
* No unnecessary white lines